robottelo/cli/factory.py

```diff
@@ -321,6 +325,115 @@ def make_content_view(options=None):
 
 
 @cacheable
+def make_content_view_filter(options=None):
+    """
+    Usage::
```

Comment:
anyway, it is docstring duplication as you described all that information in base class anyway. Please, choose one location only